src/components/utils/src/file_system.cc

```diff
@@ -86,7 +86,7 @@ uint64_t file_system::GetAvailableDiskSpace(const std::string& path) {
 
 uint32_t file_system::FileSize(const std::string &path) {
 #ifdef OS_WIN32
-	return 1024 * 1024;
+  return 1024 * 1024:
```

Comment:
- Please change this line code with end of semicolon ";"
- Please add necessary comment why here should be 1024*1024